meetuppizza/tests.py

```diff
@@ -8,3 +25,50 @@ def test_landing_page_is_there(self):
   def test_page_contains_pizza(self):
     response = self.client.get('/')
     self.assertContains(response, "pizza")
+
+  def test_signup_redirects(self):
+    response = self.client.post('/sign_up', self.params, follow=True)
+    self.assertRedirects(response, '/')
+
+  def test_user_is_created(self):
+    self.client.post('/sign_up', self.params)
+    self.assertEqual(1, len(User.objects.all()))
+
+  def test_user_is_logged_in_after_signup(self):
```

Comment:
:+1: this is a better name: it describes everything you're testing in the method body.